### DISCUSSION TITLE
登录超时

### DISCUSSION THREAD

**Discussion by 17Yuns** [Mrs4s/go-cqhttp#2286] [Q&A]
几天前还可以登录现在一直超时是怎么回事![Screenshot_2023_0711_233129.png](https://github.com/Mrs4s/go-cqhttp/assets/68860562/e6173a56-644a-4808-97d3-58613317bb97)

**Comment by larknotification** [comment]
我也遇到了，你是那个协议？

**Reply by 17Yuns** [reply]
啥协议都这样，不过我解决了

**Comment by 17Yuns** [comment]
在目录新建address.txt写这个ip地址120.241.130.216:8080就解决了

**Reply by huaifengbusan** [reply]
牛的

**Reply by zhangjing-GitHub-Code** [reply]
请问这些qq服务器地址在哪里可以获取到？
@yiqikeji
